helix-core/src/test/java/org/apache/helix/monitoring/mbeans/TestDropResourceMetricsReset.java

```diff
@@ -34,11 +34,15 @@
 import org.apache.helix.integration.manager.MockParticipantManager;
 import org.apache.helix.model.IdealState.RebalanceMode;
 import org.apache.helix.tools.ClusterSetup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.testng.Assert;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 public class TestDropResourceMetricsReset extends ZkUnitTestBase {
+  private static final Logger LOG = LoggerFactory.getLogger(TestDropResourceMetricsReset.class);
```

Comment:
This is general feedback. I'm not sure if we need to add logger logs here necessarily. What I meant is in order to debug this _locally_, you can add print statements to help you understand what's happening behind the scenes. That doesn't mean that you need to add logs in the test. So I'm not sure if this commit is really a value-add because this commit doesn't contain anything that fixes the test.

Reply:
The test never succeeds in the local environment. Even prior to ZK version bump-up, the test fails locally for me. I went back to really old CL# and only difference where it used to work constantly and now failing intermittently is related to one additional message being sent by ZK when participant node 'syncStop' happens. So that was not enough for me to debug. I wanted to see how it passes in CI pipeline. 
If we want to resolve intermittent failures in CI, debug messages are required.